app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_database = {}
+
+    def wrapper(*args) -> Any:
+        if tuple(args) not in cache_database:
```

Comment:
not need to use `tuple` 

Please fix it here and other places 
```suggestion
        if args not in cache_database:
```